### ISSUE TITLE
[Bug] broadcast无法指定tmp_ub的地址，在复用ub时会比较麻烦

### ISSUE THREAD

**Issue by liuwenda4** [tile-ai/tilelang-ascend#1221]
## Bug Description
broadcast无法指定tmp_ub的地址，在复用ub时会比较麻烦,在前端只有dst和src，还有axis，没有ub地址的参数，然而ascendc有这个选项
## Environment
A2，A3
**Hardware:**
- NPU Device: [e.g. Ascend A2, A3]
- Other hardware info: [e.g. CPU, memory]

**Software:**
- OS: [e.g. Ubuntu 20.04, EulerOS]
- CANN version: [e.g. 8.3.RC1]
- torch-npu version: [e.g. 2.6.0.RC1]
- Python version: [e.g. 3.10]
- TileLang-Ascend version: [e.g. commit hash or tag]
- Installation method: [wheel package / build from source]

## Code to Reproduce

```python
# Minimal reproducible code
import tilelang
import torch

# Your code here
```

## Error Message

```
Paste the full error message here
```

## Expected Behavior
broadcast可以指定tmp_ub
## Actual Behavior
A clear and concise description of what actually happened.
broadcast没有这个tmp_ub的选项
## Additional Context

**Programming Mode:**
- [x ] Developer mode (automatic sync/buffer management)
- [x ] Expert mode (manual T.Scope/T.barrier_all)
- [x ] Mixed mode

**Backend:**
- [x ] Ascend C & PTO backend
- [ ] AscendNPU IR backend

**Pass Configs (if applicable):**
```python
pass_configs = {
    tilelang.PassConfigKey.TL_ASCEND_AUTO_SYNC: ...,
    tilelang.PassConfigKey.TL_ASCEND_MEMORY_PLANNING: ...,
    # ...
}
```

**Operator Type:**
[e.g. GEMM, Flash Attention, Softmax, Convolution, etc.]

**Additional Information:**
- Have you checked the [Programming Guide](../docs/TileLang-Ascend%20Programming%20Guide.md)?
- Have you searched existing issues?
- Any other context or screenshots?

**Comment by ArmandAlbert** [comment]
**There`s a same ISSUE #1164 about the application of tmp buffer.**  Currently, a feasible solution is to modify pass and support manual allocation of the tmp buffer.

**Comment by ArmandAlbert** [comment]
PR https://github.com/tile-ai/tilelang-ascend/pull/1256 has added this feature. Please check.
